polaris.shopify.com/scripts/get-props/yarn.lock

```diff
@@ -1,2305 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
`polaris.shopify.com/scripts/get-props` has [its own `package.json`](https://github.com/Shopify/polaris/blob/main/polaris.shopify.com/scripts/get-props/package.json) (for reasons I'm not entirely certain, but I _think_ it's because it needed a version of a dep that's incompatible with the workspace's version), how will `pnpm` behave in this situation? Do we need to pull the script up into its own workspace so it can perform the same function?

Reply:
Yeah, let's do it! Cleaning up is always better than perpetuating workarounds 🎉

Reply:
Yep, #1 sounds like the correct approach here - sounds like it was a workaround due to yarn's hoisting. pnpm doesn't have that problem, so let's just push it up to the `polaris.shopify.com/package.json` :+1:

Reply:
Agree on moving forward with option 1 👍

Reply:
Option 1 didn't work 😓 

Have reverted to a nested workspace and pulled this out into an issue to tackle later: https://github.com/Shopify/polaris/issues/11863